lib/matplotlib/textpath.py

```diff
@@ -429,3 +430,23 @@ def _revalidate_path(self):
             self._cached_vertices = tr.transform(self._vertices)
             self._cached_vertices.flags.writeable = False
             self._invalid = False
+
+    def __deepcopy__(self, memo):
+        """Update path and create deep copy."""
+        self._revalidate_path()
+        cls = self.__class__
+        new_instance = cls.__new__(cls)
+        memo[id(self)] = new_instance
```

Comment:
I don't think you need to fill in the memo yourself, `copy.deepcopy` already does that for you: https://github.com/python/cpython/blob/7c2a040a10654d67ff543a55858ba2d7a9f7eea8/Lib/copy.py#L175-L176